File: src/main/java/com/gangofconnectfour/powerfourservice/repository/SoloGameRepository.java
package com.gangofconnectfour.powerfourservice.repository;

import com.gangofconnectfour.powerfourservice.model.SoloGame;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface SoloGameRepository extends JpaRepository<SoloGame, Long> {
}

File: src/main/kotlin/com/gangofconnectfour/powerfourservice/repository/SoloGameRepository.kt
package com.gangofconnectfour.powerfourservice.repository

import com.gangofconnectfour.powerfourservice.model.SoloGame
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface SoloGameRepository : JpaRepository<SoloGame, String> {
}